modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/AbstractPhpCodegen.java

```diff
@@ -218,7 +218,7 @@ public void processOpts() {
         // supportingFiles.add(new SupportingFile("LICENSE", "", "LICENSE"));
 
         // all PHP codegens requires Composer, it means that we need to exclude from SVN at least vendor folder
-        supportingFiles.add(new SupportingFile(".gitignore", "", ".gitignore"));
```

Comment:
I found that `.gitignore` files sometimes don't load via `this.getClass().getClassLoader().getResource(path)`.

For example, this works: `this.getClass().getClassLoader().getResource("_common/.openapi-generator-ignore")` while this did not: `this.getClass().getClassLoader().getResource("graphql-nodejs-express-server/.gitignore")`. This only seems to affect PHP and the GraphQL Express servers as everything else is either `gitignore` or `gitignore.mustache` (the formats documented as acceptable on Java documentation).